site/docs/v1/tech/themes/tailwind.adoc

```diff
@@ -0,0 +1,208 @@
+THEME_ID=5d175a65-71f2-489a-a6d2-89adad4f5b00 # your theme's UUID
+FUSIONAUTH_URL=http://localhost:9011 # your FusionAuth server, including the protocol and hostname, but no paths
+TMP_DIR=src # a temporary directory; this will be created
+```
+
+To initially download the theme from FusionAuth, run the following command:
+```bash
+./download.sh
+```
+
+Next, install the dependencies and add tailwindcss:
+```bash
+npm install
+npm install tailwindcss@latest
+npx tailwindcss init
+```
+
+This will install Tailwind CSS and create a `tailwind.config.js` file in the project root. You can customize the configuration as needed.
+
+Change the `tailwind.config.js` to handle the FreeMarker template files in the `src` directory:
+```js
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./src/*.ftl'],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
+```
+
+Create a new stylesheet `input.css` in the root directory:
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+This stylesheet will be used to import the Tailwind CSS base, components, and utilities. Additionally, you can add custom classes to this stylesheet as well.
+
+Currently, the theme is using the `fusionauth-style.css` hosted in the FusionAuth instance. This could conflict with our Tailwind CSS file. To fix this, we need to remove the following line from the `src/helpers.ftl` file - it should be around line number 85:
+```html
+  <link rel="stylesheet" href="/css/fusionauth-style.css?version=${version}"/>
+```
+
+Finally, we add a new script to the `package.json` file:
+```json
+"scripts": {
+  "watch": "node --dns-result-order ipv4first ./startwatch.js",
+  "watch:tailwind": "npx tailwindcss -i ./input.css -o ./src/styles.css --watch"
+},
+```
+
+This script will build the Tailwind CSS file and output it to the `src` directory.
+
+At this point, we have prepared the project to build the Tailwind CSS file and upload the theme to FusionAuth. To apply the changes in the `helper.ftl` file, run the following command:
+```bash
+./upload.sh
+```
+
+== Usage
+
+There are now two scripts that can be used to build the Tailwind CSS file and upload the theme to FusionAuth:
+
+* `npm run watch` — watch the template directory for changes and upload the theme to FusionAuth
+* `npm run watch:tailwind` — watch the template directory for changes and rebuild the Tailwind CSS file
+
+If you start both scripts in separate terminal windows, you can edit the template files, which rebuild the Tailwind CSS file, and automatically upload the theme to FusionAuth.
+
+Now if we use a Tailwind CSS class in a template, it will be included in the CSS file:
+```html
+<div class="bg-gray-200">
+  <h1 class="text-2xl">Hello World</h1>
+</div>
+```
+
+To preview the changes, open FusionAuth, navigate to [breadcrumb]#Customization -> Themes#. Choose your theme, then click [uielement]#Preview# (the green magnifying glass).
+
```

Comment:
Can you please put an image of the preview screen and/or the page where you'd click the preview button here?